### PR TITLE
rec: followup to #9654: Add Policy Kind / RPZ action to Protobuf messages

### LINKED ISSUE #9653: Add RPZ action to Protobuf message
- Program: Recursor
 - Issue type: Feature request

### Short description
In https://github.com/PowerDNS/pdns/issues/9058, the RPZ protobuf message was extended to include the RPZ trigger.
It would be useful to also add the Policy Kind (RPZ action) to the messages, similarly to what's done here:
https://github.com/PowerDNS/pdns/blob/rel/rec-4.4.x/pdns/filterpo.cc#L553

### Usecase
This would allow reviewing the log of a policy hit to understand what action was taken.

I will submit a pull request for the feature.

### PR DESCRIPTION
**Pull request by omoerbeek**
### Short description

 Followup up #9654, git does not let me push to the original branch: https://github.com/threatstop/pdns master 

Fixes  #9653

Will also add regression test.

### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled this code
- [X] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [X] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: regression-tests.recursor-dnssec/test_Protobuf.py [hunk 1]
```diff
@@ -931,7 +933,7 @@ def testA(self):
         # then the response
         msg = self.getFirstProtobufMessage()
         self.checkProtobufResponse(msg, dnsmessage_pb2.PBDNSMessage.UDP, res)
-        self.checkProtobufPolicy(msg, dnsmessage_pb2.PBDNSMessage.PolicyType.QNAME, 'zone.rpz.', '*.test.example.', 'sub.test.example')
+        self.checkProtobufPolicy(msg, dnsmessage_pb2.PBDNSMessage.PolicyType.QNAME, 'zone.rpz.', '*.test.example.', 'sub.test.example', dnsmessage_pb2.PBDNSMessage.PolicyKind.NoAction)
```
Comment (rgacogne) [inline regression-tests.recursor-dnssec/test_Protobuf.py:936]: Not your fault at all but I'm a bit sad we only have `NoAction` examples here.. :-/

**Review by rgacogne — APPROVED** [review]
Looks good to me! Are the `builder` changes expected?

**Comment by omoerbeek** [comment]
> Looks good to me! Are the `builder` changes expected?

Nope, `git add -u` accident

File: regression-tests.recursor-dnssec/test_Protobuf.py (hunk 1 above)
Comment (omoerbeek) [reply regression-tests.recursor-dnssec/test_Protobuf.py:936]: Yes, will make a note to add some more one day.

**Pull request merged**